shell_test.go

```diff
@@ -10,11 +10,12 @@ import (
 	"time"
 
 	"github.com/cheekybits/is"
+	"github.com/ipfs/go-ipfs-api/options"
 )
 
 const (
 	examplesHash = "QmS4ustL54uo8FzR9455qaxZwuMiUhyvMcX9Ba8nUH4uVv"
-	shellUrl     = "localhost:5001"
+	shellUrl     = "https://ipfs.infura.io:5001"
```

Comment:
I'd leave `localhost:5001`

Reply:
Oops, ya, I didn't mean to make a pr here just yet. Will definitely switch that back. 

Having problems running tests on my computer all of a sudden though. Any idea what could be causing these errors when I run `go test`?

```shell_test.go:26:  != QmUfZ9rAdhV5ioBzXKdUTh2ZNsz9bzbkaLVyQ8uc8pj21F
--- FAIL: TestAdd (0.04s)
        shell_test.go:41: unexpected nil: (*shell.Shell)(nil)
--- FAIL: TestLocalShell (0.00s)
        shell_test.go:93: expected nil: &shell.Error{Command:"file/ls", Message:"must be passed a buffer", Code:1}
--- FAIL: TestFileList (0.01s)
        shell_test.go:129: expected true: false
--- FAIL: TestPins (0.07s)```

Reply:
Suspect it might be failing to connect to by local node, but it says `API is listening on: /ip4/127.0.0.1/tcp/5001`

Reply:
Suspect it might be failing to connect to my local node but this was printed after running the daemon: `API is listening on: /ip4/127.0.0.1/tcp/5001`

Reply:
Looks like different errors get thrown when the node isn't running though.